src/switch_core.c

```diff
@@ -1852,6 +1857,7 @@ SWITCH_DECLARE(switch_status_t) switch_core_init(switch_core_flag_t flags, switc
 	runtime.min_dtmf_duration = SWITCH_MIN_DTMF_DURATION;
 	runtime.odbc_dbtype = DBTYPE_DEFAULT;
 	runtime.dbname = NULL;
+	runtime.add_media_bug_last = SWITCH_TRUE;
```

Comment:
out of curiosity - could this be implemented as another flag in `runtime`? then check if via `switch_core_flags()`?